### PR TITLE
Ci 144 update capital invest landing page

### PR DESCRIPTION
**Pull request by jessgilbert**

### CODE REVIEW

File: great_international/serializers.py [hunk 1]
```diff
@@ -308,6 +258,37 @@ class RelatedCapitalInvestOpportunityPageSerializer(BasePageSerializer):
...
+    def get_related_region(self, obj):
+        serialized = []
+        region = obj.related_region
+
+        if not region:
+            return serialized
+        serializer = RelatedCapitalInvestPageSerializer(
+            region.specific)
+        serialized.append(serializer.data)
+        return serialized[0]
```
Comment (sdonk) [inline great_international/serializers.py]: Can you not just return `serializer.data`?

Comment (jessgilbert) [reply great_international/serializers.py]: I can indeed you're completely right, will change them :)

File: great_international/models.py [hunk 2]
```diff
@@ -1718,207 +1718,183 @@ class InternationalEUExitFormSuccessPage(ExclusivePageMixin, BasePage):
...
-        max_length=255,
-        blank=True
-    )
-    reason_to_invest_section_intro = models.CharField(
-        max_length=255,
-        blank=True
-    )
-    reason_to_invest_section_content = MarkdownField(blank=True)
-    reason_to_invest_section_image = models.ForeignKey(
+class RegionCardField(models.Model):
```
Comment (sdonk) [inline great_international/models.py]: Any particular reason why this is not inheriting from `BasePage`

File: great_international/models.py [hunk 3]
```diff
@@ -1718,207 +1718,183 @@ class InternationalEUExitFormSuccessPage(ExclusivePageMixin, BasePage):
...
+        related_name='added_region_card_fields',
         blank=True,
-        on_delete=models.SET_NULL,
-        related_name='+'
+        null=True,
     )
-    region_card_two_image = models.ForeignKey(
+
+
+class HomesInEnglandCardField(models.Model):
```
Comment (sdonk) [inline great_international/models.py]: ditto

**Pull request merged**